spikeinterface/sorters/utils/misc.py

```diff
@@ -1,3 +1,11 @@
+from typing import List, Union
+
+import numpy as np
+import spikeinterface as si
+from spikeinterface.core import (BaseRecording, BaseSorting,
+                                 BaseRecordingSegment, BaseSortingSegment)
+import spikeextractors as se
```

Comment:
Please don't import spikeextractors at global level because it make a necessary dependency.
Import it in function or class when necessary.

Reply:
Got it. How about just not importing spikeextractors? We assume we are starting with an old recording extractor which means the user has imported the older spikeextractors already.